src/generate-font-faces.js

```diff
@@ -16,6 +16,6 @@ export default function generateFontFaces(fontDictionary) {
 function asFontFaceSrc(urls) {
   // `urls` is a dictionary of font types (woff, woff2 etc) to url string
   return Object.keys(urls).map(
-    type => `url("/_static/${urls[type]}") format("${type}")\n`
```

Comment:
Is there no way to add a test for this? This was a bug so we should somehow make sure it doesn't happen again.

Reply:
It wasn't really a bug, it was compensating for assetURL not working correctly — to make the path correct. If the path is wrong the browser will let us know.